mathml/presentation-markup/mpadded/mpadded-002.html

```diff
@@ -126,6 +126,11 @@
       <mpadded id="height0" height="25px"><mspace width="100px" depth="125px" height="150px" style="background: black; opacity: .5"/></mpadded>
       <mpadded id="height1" height="50px"><mspace width="100px" depth="125px" height="150px" style="background: black; opacity: .5"/></mpadded>
       <mpadded id="height2" height="75px"><mspace width="100px" depth="125px" height="150px" style="background: black; opacity: .5"/></mpadded>
+    </math>
+  </p>
+  <p>
+    <span id="baseline" style="display: inline-block; width: 30px; height: 5px; background: blue"></span>
```

Comment:
it seems wrong to have two elements with "baseline" id, shouldn't the JS part be updated too?

Reply:
Right, good catch! Fixed.